nvflare/apis/resource_manager_spec.py

```diff
@@ -26,12 +24,11 @@ def consume(self, resources: dict):
 
 class ResourceManagerSpec(ABC):
     @abstractmethod
-    def check_resources(self, resource_requirement: dict, fl_ctx: FLContext) -> (bool, Optional[str]):
+    def check_resources(self, resource_requirement: dict) -> (bool, Optional[str]):
```

Comment:
It's still better to add fl_ctx: FLContext back into the API. Because we are providing an API spec for people to write their own ResourceManager implementation. We are not using the fl_ctx in own standard implementation. But other people could make use of this for their own purpose. By providing the fl_ctx: FLContext in the API signature, it provides a lot more flexibility.